Competition/.vscode/settings.json

```diff
@@ -14,5 +14,68 @@
     "**/.factorypath": true,
     "**/*~": true
   },
-  "C_Cpp.default.configurationProvider": "vscode-wpilib"
+  "C_Cpp.default.configurationProvider": "vscode-wpilib",

```

Comment:
I don't know if we should check this in....
This seems user specific?

Reply:
yeah...i have no clue what this is

Reply:
should i just delete it or is there a way to not add the changes to this file

Reply:
mehhh it should be fine. I dont really want to deal with it lol. we can leave it